script/parcel

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+in="js/index.js"
+out="resources-compiled/public/js/"
+
+echo "parcel $1 $in -d $out"
+parcel $1 "$in" -d "$out"
```

Comment:
One alternative common in JS world would be to define commands such `watch` and `build` in `"scripts"` property in package.json and invoke them by e.g. `npm run build`, see https://docs.npmjs.com/cli/run-script